content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -224,6 +224,55 @@ If you are not running kube-proxy on a host running the API server, then you mus
 
 {{% /capture %}}
 
+### Register APIService objects
+
+You can dynamically configure what client requests are proxied to extension
+apiserver. The following is an example registration:
+
+```yaml
+
+apiVersion: apiregistration.k8s.io/v1beta1
```

Comment:
use `apiregistration.k8s.io/v1`? also in L263

Reply:
Yes, use v1 types wherever we have them